test/api/CMakeLists.txt

```diff
@@ -1,20 +1,21 @@
 
-function(add_api_test TEST_NAME TEST_SRC)
+function(add_api_test TEST_NAME TEST_SRC TEST_DIR)
     set(NAME test_api_${TEST_NAME})
     add_executable(${NAME} EXCLUDE_FROM_ALL ${TEST_SRC})
     rocm_clang_tidy_check(${NAME})
     target_link_libraries(${NAME} migraphx_c)
     target_include_directories(${NAME} PUBLIC ../include)
-    add_test(NAME ${NAME} COMMAND $<TARGET_FILE:${NAME}> WORKING_DIRECTORY ${TEST_ONNX_DIR}) 
+    add_test(NAME ${NAME} COMMAND $<TARGET_FILE:${NAME}> WORKING_DIRECTORY ${TEST_DIR}) 
     add_dependencies(tests ${NAME})
     add_dependencies(check ${NAME})
 endfunction()
 
 
-add_api_test(ref test_cpu.cpp)
-add_api_test(save_load test_save_load.cpp)
-add_api_test(op test_op_construct.cpp)
+add_api_test(ref test_cpu.cpp ${TEST_ONNX_DIR})
+add_api_test(save_load test_save_load.cpp ${TEST_ONNX_DIR})
+add_api_test(op test_op_construct.cpp ${TEST_ONNX_DIR})
+add_api_test(parser test_parser.cpp ${TEST_TF_DIR})
```

Comment:
Maybe this should be called `test_tf_parser.cpp` instead since it just tests parse_tf.

Reply:
fixed